src/amo/components/SearchSort/SearchSort.scss

```diff
@@ -0,0 +1,82 @@
+@import '~core/css/inc/mixins';
+@import '~core/css/inc/vars';
+
+@mixin arrow-link-right-aligned() {
+  background-image: url('~amo/img/icons/arrow-link-color.svg');
+  margin-right: 0;
+  position: absolute;
+  right: 8px;
+  top: 14px;
+}
+
+.SearchSort {
+  background: $base-color;
+  border-radius: $border-radius-default;
+  border: 0;
+  margin: 0 5% 10px;
+  overflow: hidden;
+}
+
+.SearchSort-toggle {
```

Comment:
Yeah, eventually moving this out would be nice but I think that's a refactor we can do later, if that's cool?